scripts/lib/html_normalizer.py

```diff
@@ -123,11 +131,64 @@ def flag_potential_footnotes(
     return html
 
 
-def normalize(html: str, url: str) -> str:
+def get_image_size(desc: re.Match) -> dict:
+    for attrs in desc["attrs"], desc["wrap"]:
+        if not attrs:
+            continue
+
+        for size_re in IMG_RE["size_attr"], IMG_RE["size_style"]:
+            sizes = dict(size_re.findall(attrs))
+
+            if not len(sizes):
+                continue
+
+            for k in sizes:
+                if sizes[k][-1].isnumeric():
```

Comment:
What if not numeric?

Reply:
if it is not numeric, we already have a unit in the string

Reply:
Perhaps good to add that as a comment